crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/config/AuthorizationServerConfiguration.java

```diff
@@ -26,14 +26,15 @@
 import java.security.KeyPairGenerator;
 import java.security.interfaces.RSAPrivateKey;
 import java.security.interfaces.RSAPublicKey;
-import java.util.UUID;
 
 @Configuration(proxyBeanMethods = false)
 @Order(2)
 public class AuthorizationServerConfiguration {
     @Value("${means-assessment.security.issuer-uri}")
     private String issuerUri;
 
+    private static final String KEY_ID = "c097657e-4111-4e88-a8d5-d70c9c27ba53c097657e-4111-4e88-a8d5-d70c9c27ba53";
```

Comment:
I think the KeyID can be safely removed.

It's an optional header claim which holds an identifier, and seems to be particularly useful when you have multiple keys to sign the tokens and need to look up the right one to verify the signature. However, since we only have a single key that we use to sign the tokens removing it should have no effect.